src/tasks/AppleAppBuilder/Templates/runtime.m

```diff
@@ -196,7 +201,7 @@
 //%DllMap%
 }
 
-#if FORCE_INTERPRETER || FORCE_AOT || !TARGET_OS_SIMULATOR
```

Comment:
I don't think we can just remove this, it'd not enable AOT on e.g. iOS device builds since they don't set FORCE_AOT.
You can replace it with `|| (!TARGET_OS_SIMULATOR && !TARGET_OS_MACCATALYST)`

Reply:
I removed the TARGET_OS_SIMULATOR stuff on the basis I couldn't see where we were setting it - it didn't occur to me it came from Apple headers